code/Player/Controllers/HumanController.cs

```diff
@@ -103,6 +126,14 @@ protected virtual void UpdateBBox()
 			maxs = new Vector3( +BodyGirth, +BodyGirth, BodyHeight ) * Pawn.Scale;
```

Comment:
Not your modification but somebody know why the '+' ?

Reply:
To be explicit about the sign I think